scripts/binary/build.ts

```diff
@@ -0,0 +1,331 @@
+import os from 'os'
+import fs from 'fs-extra'
+import path from 'path'
+import _ from 'lodash'
+import del from 'del'
+import chalk from 'chalk'
+import electron from '@packages/electron'
+import la from 'lazy-ass'
```

Comment:
I generally prefer more verbose names for clarity. In this case maybe `lazyAssessment`?

```suggestion
import lazyAssessment from 'lazy-ass'
```

Reply:
Yeah this was pretty much just a `js -> ts` find/replace of the existing binary build code for the most part https://github.com/cypress-io/cypress/blob/002fbdda600aa83fce90dfa2d687bbf8aae6aa87/scripts/binary/build.js#L1-L14

If we're going to change that, then maybe we wait to do it separately as a global change in the codebase?

<img src="https://user-images.githubusercontent.com/154748/142651079-803d80b1-5a05-4a47-b0f0-579c093961e5.png" width=300 />

Reply:
It would make sense to me to just change it everywhere separately.